docs/releases/5.1.txt

```diff
@@ -261,6 +261,9 @@ Models
 * The new ``"transaction_mode"`` option is now supported in :setting:`OPTIONS`
   on SQLite to allow specifying the :ref:`sqlite-transaction-behavior`.
 
+* Calling :meth:`.Model.full_clean` on a model with a field that has a
+  ``db_default`` and no value specified no longer raises a ``ValidationError``.
```

Comment:
This is not a new feature, it should go to the _"Backwards incompatible changes in 5.1"_.